### ISSUE TITLE
No output MMD File for some PDFs

### ISSUE THREAD

**Issue by BiEchi** [facebookresearch/nougat#30]
Greetings! There seems no output files when I apply Nougat to some documents. The output looks like this:

```
$ nougat ch-01.pdf -o .
...
WARNING:root:Found repetitions in sample 2
WARNING:root:Found repetitions in sample 3
WARNING:root:Skipping page 11 due to repetitions.
WARNING:root:Skipping page 12 due to repetitions.
 50%|█████████████████████████████████████                                     | 3/6 [01:16<01:06, 22.18s/it]WARNING:root:Found repetitions in sample 3
WARNING:root:Skipping page 16 due to repetitions.
100%|██████████████████████████████████████████████████████████████████████████| 6/6 [02:28<00:00, 24.71s/it]
```

What does these error messages mean? Is there any way I can successfully get an MMD file? Looking forward to your reply!

**Comment by lukas-blecher** [comment]
Hello,
the model can not process your documents because they probably differ too much from the training domain. Please note that nougat only works with english and a few related languages. 
Are the documents in a different language than english? If not could you share a page with me so I can look into it?

**Comment by BackMountainDevil** [comment]
So if I want to use it on Simple-Chinese(zh-cn), how should I do? train on Chinese dataset?

**Comment by lukas-blecher** [comment]
> I'm not the original poster, but I've successfully converted about 15 long documents now, so I know what to expect.
> 
> But on 3 of these conversions the **mmd file did not get created** despite clearly successfully converting hundreds of pages of the document.
> 
> Possibly, if nougat skips a page at/near the end of the document the output to mmd doesn't happen?

You're right. Should be fixed in [35575f0](https://github.com/facebookresearch/nougat/commit/35575f02024436b14c846d43c0601d5b15dae66a)
